epochStart/bootstrap/process.go

```diff
@@ -1179,6 +1179,7 @@ func (e *epochStartBootstrap) createRequestHandler() error {
 		ResolverConfig:              e.generalConfig.Resolvers,
 		PeersRatingHandler:          disabled.NewDisabledPeersRatingHandler(),
 		PayloadValidator:            payloadValidator,
+		IsSyncedLiteObserver:        false,
```

Comment:
:+1: in bootstrapping phase we should be able to respond to others for parts of the trie we might have.